From 39b10628c5424ab913a27f90eb05b145738f9831 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:24:32 -0400
Subject: [redhat] kabi: Adding symbol param_set_int (kernel/params.c)

Message-id: <6f5748cdfc475052858008d6c2ff878103f6f3aa.1410510773.git.poros@redhat.com>
Patchwork-id: 96134
O-Subject: [PATCH RHEL7.1 152/233] kabi: Adding symbol param_set_int (kernel/params.c)
Bugzilla: 1071098
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1071098
    https://bugzilla.redhat.com/show_bug.cgi?id=1071098

    Description:
    ============
    Not used directly, used by module_param internally for type int.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/param_set_int
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/param_set_int
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/param_set_int

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/param_set_int b/redhat/kabi/kabi-module/kabi_ppc64/param_set_int
new file mode 100644
index 0000000..4234bf3
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/param_set_int
@@ -0,0 +1,2 @@
+#1-
+0x83c8a355 param_set_int vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/param_set_int b/redhat/kabi/kabi-module/kabi_s390x/param_set_int
new file mode 100644
index 0000000..4234bf3
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/param_set_int
@@ -0,0 +1,2 @@
+#1-
+0x83c8a355 param_set_int vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/param_set_int b/redhat/kabi/kabi-module/kabi_x86_64/param_set_int
new file mode 100644
index 0000000..4234bf3
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/param_set_int
@@ -0,0 +1,2 @@
+#1-
+0x83c8a355 param_set_int vmlinux EXPORT_SYMBOL
-- 
1.7.1